### PR TITLE
Colorized output for cpu temp and battery info

### PR DESCRIPTION
**Pull request by mfulz**
With this patch you're able to change the colors for cpu_temperature and battery with using color_good directives.

### CODE REVIEW

**Comment by stapelberg** [comment]
Please don’t change unrelated files, and please squash your fixup commits. Let me know when you’ve gotten the pull request to pass travis and i’ll take another look.

File: src/pulse.c [hunk 1]
```diff
@@ -21,7 +22,8 @@ static pa_context *context = NULL;
 static pa_mainloop_api *api = NULL;
 static bool context_ready = false;
 static uint32_t default_sink_idx = DEFAULT_SINK_INDEX;
-TAILQ_HEAD(tailhead, indexed_volume_s) cached_volume =
+TAILQ_HEAD(tailhead, indexed_volume_s)
```
Comment (Airblader) [inline src/pulse.c]: @stapelberg If this is what a recent clang does with the current configuration, we should probably still change the configuration.

Comment (mfulz) [inline src/pulse.c]: I'll cleanup the code and hell fixing the formatting.
I just wanted top know, if the feature would be accepted before putting top mich work Ingo it.
I've used clang-format Version 3.7 for the formatting.

**Comment by mfulz** [comment]
I switched to i3blocks -> no need for me.

**Pull request closed without merging**